runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZoneConfig.java

```diff
@@ -87,11 +87,33 @@ default boolean overloadNotification()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "earlyoverloadwarning",
+		name = "Early overload warning",
+		description = "Toggles notifications when your overload is nearly finished",
+		position = 6
+	)
+	default boolean earlyOverloadWarning()
```

Comment:
I feel like this could be removed, and the overload warning could be disabled by being set to `0`. That would be a more concise way to add a new setting, and would be consistent with how some other plugins are implemented.